src/fiatconnect/ReviewScreen.test.tsx

```diff
@@ -43,13 +51,11 @@ function getProps(flow: CICOFlow, withFee = false, cryptoType = CryptoType.cUSD)
 }
 
 describe('ReviewScreen', () => {
-  const store = createMockStore()
```

Comment:
just for my understanding. Is this created in every test so we have a fresh set of actions or something else? If it's just for actions, I think we can also do `store.clearActions()` in a `afterEach`